README.md

```diff
@@ -159,9 +159,9 @@ To run tests, run the following command:
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215242489-c4b5d3ad-5718-4930-9bf8-466741f9a0e5.png)

- [ ] Instead of this long command **bin/rails test test/models/article_test.rb** Kindly put **rspec** because this long command is only testing your article model but this short command **rspec** test all models, controllers and views :+1: